src/runner/browser-env/vite/utils.ts

```diff
@@ -27,3 +39,39 @@ export const getNodeModulePath = async ({
 export const prepareError = (error: Error): Error => {
     return JSON.parse(JSON.stringify(error, Object.getOwnPropertyNames(error)));
 };
+
+export const getTestInfoFromViteRequest = (req: Connect.IncomingMessage): TestInfoFromViteRequest | null => {
+    if (!req.url?.endsWith("index.html") || !req.originalUrl) {
+        return null;
+    }
```

Comment:
How it behaves with query(`"?"`) or hash location(`"#"`) parameters?
And what is the difference between "url" and "originalUrl"?

Reply:
> How it behaves with query("?") or hash location("#") parameters?

It works correctly with all of these parameters. Because in one of my plugin I return only `index.html`. It means url inside `req.url` always has `/index.html`. Url inside `originalUrl` has url which is open in test. Looks like - `/run-uuids/1d344b36-a09a-4cc2-9ff9-158e91fa70a1`